mock/py/mockchain.py

```diff
@@ -138,7 +138,9 @@ def add_local_repo(infile, destfile, baseurl, repoid=None):
 
     try:
         with open(infile) as f:
-            code = compile(f.read(), infile, 'exec')
+            content = f.read()
+            content = re.sub(r'include\((.*)\)', r'exec(open(\g<1>).read(), {}, locals())', content)
+            code = compile(content, infile, 'exec')
```

Comment:
Why you remove the function and rather duplicate the code on two places?

Reply:
the code are in 2 places , just must more simplified

Reply:
I see, looking again -- the duplication comes from different reason (mockchain support) ... so wouldn't be `code = parse_config(filename)` function cleaner?